Assets/MixedRealityToolkit.SDK/Features/Input/Handlers/EyeTrackingTarget.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using UnityEngine.Events;
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    /// <summary>
+    /// A game object with the "Target" script attached can be selected and may react to hover states.
+    /// </summary>
+    public class EyeTrackingTarget : BaseEyeFocusHandler, IMixedRealityPointerHandler, IMixedRealitySpeechHandler
+    {
+        [SerializeField]
```

Comment:
nit: serialized fields should have tooltips

Reply:
https://github.com/Microsoft/MixedRealityToolkit-Unity/issues/3767